src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -231,62 +227,77 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
             throw new CommandLineException.BadArgumentValue(
-                    String.format("The read filter(s) are specified more than once: %s",
-                            Utils.join(", ", duplicateUserFilterNames)));
+                    String.format("The read filter(s) are enabled more than once: %s",
+                            Utils.join(", ", duplicateUserEnabledFilterNames)));
         }
 
-        // throw if any disabled filter is duplicated
-        final Set<String> duplicateDisabledFilterNames = Utils.getDuplicatedItems(disableFilters);
-        if (!duplicateDisabledFilterNames.isEmpty()) {
+        // throw if a filter is *disabled* more than once by the user
+        final Set<String> duplicateDisabledUserFilterNames = Utils.getDuplicatedItems(userDisabledReadFilterNames);
+        if (!duplicateDisabledUserFilterNames.isEmpty()) {
             throw new CommandLineException.BadArgumentValue(
-                    String.format("Disabled read filter(s) are specified more than once: %s",
-                            Utils.join(", ", duplicateDisabledFilterNames)));
+                    String.format("The read filter(s) are disabled more than once: %s",
+                            Utils.join(", ", duplicateDisabledUserFilterNames)));
         }
 
-        // throw if any filter is both enabled *and* disabled by the user
-        final Set<String> enabledAndDisabled = new HashSet<>(userReadFilterNames);
-        enabledAndDisabled.retainAll(disableFilters);
+        // throw if a filter is both enabled *and* disabled by the user
+        final Set<String> enabledAndDisabled = new HashSet<>(userEnabledReadFilterNames);
+        enabledAndDisabled.retainAll(userDisabledReadFilterNames);
         if (!enabledAndDisabled.isEmpty()) {
             final String badFiltersList = Utils.join(", ", enabledAndDisabled);
             throw new CommandLineException(
                     String.format("The read filter(s): %s are both enabled and disabled", badFiltersList));
         }
 
-        // warn if a disabled filter wasn't enabled by the tool in the first place
-        disableFilters.forEach(s -> {
-            if (!readFilterNames.contains(s)) {
-                throw new CommandLineException.BadArgumentValue(String.format("Disabled filter (%s) do not exists", s));
+        // throw if a disabled filter doesn't exist; warn if it wasn't enabled by the tool in the first place
+        userDisabledReadFilterNames.forEach(s -> {
+            if (!allDiscoveredReadFilters.containsKey(s)) {
+                throw new CommandLineException.BadArgumentValue(String.format("Disabled filter (%s) does not exist", s));
             } else if (!toolDefaultReadFilters.containsKey(s)) {
                 logger.warn(String.format("Disabled filter (%s) is not enabled by this tool", s));
             }
         });
 
         // warn on redundant enabling of filters already enabled by default
```

Comment:
This will read better like "warn if a filter is both default and enabled by the user"